app/services/encrypted_attribute.rb

```diff
@@ -41,9 +41,7 @@ def decrypt(cost)
     encryptor = Pii::PasswordEncryptor.new
     decrypted = try_decrypt_with_uak(encryptor) if user_access_key.present?
     return decrypted if decrypted
-    decrypted = try_decrypt_with_all_keys(encryptor, cost)
-    return decrypted if decrypted
```

Comment:
I meant keeping lines 44 and 45 of the original and modifying line 45.

either way, if the change further down to line 60 is made, should not be an issue. `try_decrypt_with_all_keys` should only ever return a String, or raise an exception.